### PR TITLE
Add a guide on Cloudwatch dashboards

### PR DESCRIPTION
**Pull request by jmhooper**
This commit adds a guide on how to create cloudwatch dashboards using the tooling introduced in https://github.com/18F/identity-devops/commit/5e9c2fd9c6fd57524269f7c8c058a3623dadf987

### CODE REVIEW

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+As a result, Login.gov supports experimental dashboards.
+These are ephemeral dashboards which can be used to create source for long-lived
+dashboards.
+
+Terraformed dashboards are distinguisehd from experiemental dashboards by the
+prefix in the dashboard name.
+A terraformed dashboard will have a prefix which is an environment name.
+For example, a terraformed dashboard may be named `prod-sample-dashboard`.
+Conversly, an experimental dashboard will use a name that identifies the author.
+For example, `jonhooper-sample-dashboard`.
```
Comment (zachmargolis) [inline _articles/appdev-cloudwatch-dashboard-guide.md:40]: WDYT of an example variable name, unless you want your name living in the docs for a while?
```suggestion
For example, `my-example-dashboard`.
```

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+## Overview
+
+Amazon Cloudwatch makes it possible to search login.gov logs and AWS metrics to
+create visualizations or display statistics.
+Clouwatch Insights is the tool that makes it possible to query logs.
+Information on how to build Cloudwatch Insights queries can be found in the
+[AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html).
+
+]Login.gov's Clouwatch Dashboards](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#dashboards:)
```
Comment (zachmargolis) [inline _articles/appdev-cloudwatch-dashboard-guide.md:16]: flip that bracket
```suggestion
[Login.gov's Clouwatch Dashboards](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#dashboards:)
```

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+Once an experimental dashboard reaches the desired state it should be converted
+into a terraformed dashboard.
+The `identity-devops` repo has tooling for converting dashboards to terraformed
+dashboards.
+
+Given an expriemental dashboard named `jonhooper-sample-dashboard`, run the
+following in the `identity-devops` repo:
+
+```bash
+aws-vault exec prod-power -- bin/copy-cloudwatch-dashboard -i jonhooper-sample-dashboard
```
Comment (zachmargolis) [inline _articles/appdev-cloudwatch-dashboard-guide.md:51]: ditto for example names
```suggestion
Given an expriemental dashboard named `my-sample-dashboard`, run the
following in the `identity-devops` repo:

```bash
aws-vault exec prod-power -- bin/copy-cloudwatch-dashboard -i my-sample-dashboard
```

Sometimes for things intended to be replaced we could also do `$DASHBOARD_NAME` 🤷

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 4]
```diff
@@ -0,0 +1,56 @@
+---
+title: "Cloudwatch Dashboards"
+description: "How to search for, create, and manage cloudwatch dashboards"
+layout: article
+category: "AppDev"
+---
+
+## Overview
+
+Amazon Cloudwatch makes it possible to search login.gov logs and AWS metrics to
```
Comment (zachmargolis) [inline _articles/appdev-cloudwatch-dashboard-guide.md:10]:
```suggestion
Amazon Cloudwatch makes it possible to search Login.gov logs and AWS metrics to
```

**Review by zachmargolis — APPROVED** [review]
LGTM

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+description: "How to search for, create, and manage cloudwatch dashboards"
+layout: article
+category: "AppDev"
+---
+
+## Overview
+
+Amazon Cloudwatch makes it possible to search Login.gov logs and AWS metrics to
+create visualizations or display statistics.
+Clouwatch Insights is the tool that makes it possible to query logs.
```
Comment (aduth) [inline _articles/appdev-cloudwatch-dashboard-guide.md:12]: Typo

```suggestion
Cloudwatch Insights is the tool that makes it possible to query logs.
```

Comment (aduth) [reply _articles/appdev-cloudwatch-dashboard-guide.md:12]: Actually, I think the "w" is meant to be capitalized.

```suggestion
CloudWatch Insights is the tool that makes it possible to query logs.
```

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 6]
```diff
@@ -0,0 +1,56 @@
...
+Information on how to build Cloudwatch Insights queries can be found in the
+[AWS documentation](https://docs.aws.amazon.com/AmazonCloudWatch/latest/logs/CWL_QuerySyntax.html).
+
+[Login.gov's Clouwatch Dashboards](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#dashboards:)
```
Comment (aduth) [inline _articles/appdev-cloudwatch-dashboard-guide.md:16]: Same typo

```suggestion
[Login.gov's CloudWatch Dashboards](https://us-west-2.console.aws.amazon.com/cloudwatch/home?region=us-west-2#dashboards:)
```

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 7]
```diff
@@ -0,0 +1,56 @@
...
+can be found by navigating to the Cloudwatch service in the AWS Console and
+selecting "Dashboards" in the left pane.
+From here you will see a list of available dashboards and there is functionality
+for searching dashboards.
+
+## Experimental vs Terraformed Dashbords
```
Comment (aduth) [inline _articles/appdev-cloudwatch-dashboard-guide.md:22]: Typo

```suggestion
## Experimental vs Terraformed Dashboards
```

File: _articles/appdev-cloudwatch-dashboard-guide.md [hunk 8]
```diff
@@ -0,0 +1,56 @@
...
+
+The source for Cloudwatch Dashboards should be maintained in terraform code
```
Comment (aduth) [inline _articles/appdev-cloudwatch-dashboard-guide.md:24]: Capitalize?

```suggestion
The source for CloudWatch Dashboards should be maintained in Terraform code
```

**Review by aduth — APPROVED** [review]

**Pull request merged**